frontend/lib/pages/loc-confirmation.tsx

```diff
@@ -65,17 +69,31 @@ const LetterConfirmation = withAppContext((props: AppContextType): JSX.Element =
   const { letterRequest } = props.session;
   const { locPdfURL } = props.server;
 
+  let letterConfirmationPageTitle;
+
+  // I was getting a bunch of Object is possibly 'null' errors here
```

Comment:
This comment isn't really required, IMO it is implicit from the code. (Also, it's a valid use case right now because this page is actually publicly accessible and can be viewed by logged-out users, and users who haven't completed the LoC flow, which means `letterRequest` most definitely can be `null`, hehe.)

Uhm that said, I don't believe you need the `&& letterRequest.mailChoice &&` in there, you should be good with just `letterRequest && letterRequest.mailChoice == blah`?